release.env

```diff
@@ -0,0 +1,3 @@
+RELEASE_DIR="<Release dir on server>"
+RELEASE_WEB="https://raw.githubusercontent.com/DOMjudge/domjudge-scripts/master/make_release.sh"
+TOKEN="<See GitLab config>"
```

Comment:
Maybe better to move this completely out of the repo, into a file under `$HOME`.

Reply:
I think most of these constants are now removed in this PR.